test/unified_malloc_framework/memoryPool.hpp

```diff
@@ -251,6 +252,36 @@ TEST_P(umfPoolTest, multiThreadedMallocFreeRandomSizes) {
     }
 }
 
+TEST(umfMemTest, outOfMem) {
+    auto [ret, provider] =
+        umf::memoryProviderMakeUnique<umf_test::provider_mock_out_of_mem>();
+    ASSERT_EQ(ret, UMF_RESULT_SUCCESS);
+    auto hProvider = provider.get();
+
+    auto [retp, pool] =
+        umf::poolMakeUnique<umf_test::proxy_pool>(&hProvider, 1);
+    ASSERT_EQ(retp, UMF_RESULT_SUCCESS);
+    auto hPool = pool.get();
+
+    static constexpr size_t allocSize = 16;
+    static constexpr size_t maxAllocs = 9;
+
+    std::vector<void *> allocations(maxAllocs + 1, nullptr);
+
+    for (size_t i = 0; i <= maxAllocs; i++) {
```

Comment:
`maxAllocs` suggests that there is this number of allocs allowed. Please, change the condition to `i < maxAllocs`, `maxAllocs` value to 10 and the size of the `allocations` vector to `maxAllocs`.

Reply:
It's been removed for now.